### PR TITLE
Update auth.md

### PR DESCRIPTION
**Pull request by davidy718**
## Description
change a few things under okta for better understating 

## Motivation and Context
To make it easier to understating when trying to follow the steps 

## Checklist:

- [ ] My change requires a change to the documentation or CHANGELOG.
- [ ] I have updated the documentation/CHANGELOG accordingly.
- [x] I have created a feature (non-master) branch for my PR.

### CODE REVIEW

File: docs/versioned_docs/version-7.1.x/configuration/auth.md [hunk 1]
```diff
@@ -300,9 +300,10 @@ you may wish to configure an authorization server for each application. Otherwis
     skip_provider_button = true
     ```
 
-The `oidc_issuer_url` is based on URL from your **Authorization Server**'s **Issuer** field in step 2, or simply https://corp.okta.com .
+The `oidc_issuer_url` is based on URL from your **Authorization Server**'s **Issuer** field in step 2, or simply https://corp.okta.com (If your not using Authorization Server).
```
Comment (JoelSpeed) [inline docs/versioned_docs/version-7.1.x/configuration/auth.md:303]: What about this instead, it's a bit more explicit?
```suggestion
The `oidc_issuer_url` is based on the URL from your **Authorization Server**'s **Issuer** field in step 2, or if you are not using an Authorization server, the global issuer URL https://corp.okta.com.
```

**Review by JoelSpeed — COMMENTED** [review]
One quick suggestion for improved readability but otherwise LGTM

You've updated the versioned documentation, you should also update the main docs to match these changes once they're agreed otherwise the change will disappear in the future.

Please also add a changelog entry if you wish to do so

File: docs/versioned_docs/version-7.1.x/configuration/auth.md (hunk 1 above)
Comment (davidy718) [reply docs/versioned_docs/version-7.1.x/configuration/auth.md:303]: ok looks good. not sure how to accept your change's
and update the main docs

Comment (JoelSpeed) [reply docs/versioned_docs/version-7.1.x/configuration/auth.md:303]: I think you can just click commit suggestion to accept the change, or copy paste and overwrite manually

And the main docs would be in https://github.com/oauth2-proxy/oauth2-proxy/blob/master/docs/docs/configuration/auth.md, it should look pretty much identical

**Pull request closed without merging**